api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -185,6 +195,10 @@ type VRGResourceMeta struct {
 	//+optional
 	ProtectedPVCs []string `json:"protectedpvcs,omitempty"`
 
+	// List of CGs that are protected by the VRG resource
```

Comment:
Instead of adding a synthetic `name` for each group, can we simplify this to something like so:
```
  status:
    resourceConditions:
      resourceMeta:
        generation: 1
        kind: Test
        name: test
        namespace: test
        protectedpvcs:
        - mongo-1
        - mongo-2
        - logger
        - data
        pvcgroups:
        - grouped:
          - mongo-1
          - mongo-2
        - grouped:
          - logger
          - data
```

Generated using this sample change to the types file:

```
index b4cb8d04..7164b143 100644
--- a/api/v1alpha1/drplacementcontrol_types.go
+++ b/api/v1alpha1/drplacementcontrol_types.go
@@ -167,6 +167,10 @@ type PlacementDecision struct {
        ClusterNamespace string `json:"clusterNamespace,omitempty"`
 }
 
+type Groups struct {
+       Grouped []string `json:"grouped,omitempty"`
+}
+
 // VRGResourceMeta represents the VRG resource.
 type VRGResourceMeta struct {
        // Kind is the kind of the Kubernetes resource.
@@ -184,6 +188,7 @@ type VRGResourceMeta struct {
        // List of PVCs that are protected by the VRG resource
        //+optional
        ProtectedPVCs []string `json:"protectedpvcs,omitempty"`
+       PVCGroups     []Groups `json:"pvcgroups,omitempty"`
 
        // ResourceVersion is a value used to identify the version of the
        // VRG resource object
```

Reply:
@ShyamsundarR , what do you mean, when saying "a synthetic name"? It is actually the name of the CG resource, we don't want to show it?

>     protectedcgs:
>     - name: rook-ceph-dr1-1deployment-rbd-drpc
>       protectedpvcs:
>       - extra-pvc
>       - busybox-pvc